cpp/src/Ice/Communicator.cpp

```diff
@@ -30,3 +36,297 @@ Ice::Communicator::flushBatchRequestsAsync(CompressBatch compress)
         });
     return promise->get_future();
 }
+
+void
+Ice::Communicator::destroy() noexcept
```

Comment:
All this code comes from CommunicatorI.cpp. The only difference is there is no "I".